ckanext/geodatagov/commands.py

```diff
@@ -1049,30 +1049,38 @@ def jsonl_export(self):
                 time.sleep(wait_time)
                 attempts += 1
 
-            for n, dataset in enumerate(datasets):
-                os.write(fd, '%s\n' % dataset)
+            os.write(fd, json.dumps(datasets))
```

Comment:
This outputs a single array. We need the output to be JSONL, newline-separated objects. You'll want to keep the existing code, just json encode each line:

```python
for n, dataset in enumerate(datasets):
  os.write(fd, '%s\n' % json.dumps(dataset))
```